pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -471,6 +475,7 @@ type BackupSpec struct {
 	ContainerSecurityContext *corev1.SecurityContext      `json:"containerSecurityContext,omitempty"`
 	Resources                *ResourcesSpec               `json:"resources,omitempty"`
 	RuntimeClassName         *string                      `json:"runtimeClassName,omitempty"`
+	PITR                     PITRSpec                     `json:"pitr,omitempty"`
```

Comment:
According to task description
`If the user specifies multiple buckets in spec.backup.storages, then PITR will not be enabled and we are going to throw an error in the operator logs`

We should check this, and probably store the storage name since
`If the user had one bucket and pitr enabled and now adds one more bucket, PITR continues to work, we throw an error in the operator logs`

Reply:
Hey @qJkee! Validation is planned for another PR. Also, the second case is done at users own risk.

It has been discussed in slack :)